src/LiveComponent/doc/index.rst

```diff
@@ -3104,34 +3104,40 @@ on the component:
 
 .. code-block:: diff
 
-    // ...
+      // src/Components/ProductSearch.php
+      namespace App\Components;
 
-    #[AsLiveComponent]
-    class ProductSearch
-    {
-      - #[LiveProp(writable: true)]
-      + #[LiveProp(writable: true, onUpdated: 'onQueryUpdated')]
-        public string $query = '';
+      use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+      use Symfony\UX\LiveComponent\Attribute\LiveProp;
 
-        // ...
+      #[AsLiveComponent]
+      class ProductSearch
+      {
+    -     #[LiveProp(writable: true)]
+    +     #[LiveProp(writable: true, onUpdated: 'onQueryUpdated')]
```

Comment:
Instead of diff, let's make these normal PHP types. I think it's easy enough to see the `onUpdated` part. And then we will get syntax highlighting also.